streaming-common/src/test/java/com/sequenceiq/cloudbreak/streaming/processor/RoundRobinStreamProcessingQueuesTest.java

```diff
@@ -53,8 +53,8 @@ public void testProcess() throws InterruptedException {
 
     static class DummyRequest extends RecordRequest {
 
-        DummyRequest(String rawBody, GeneratedMessageV3 messageBody, long timestamp) {
-            super(rawBody, messageBody, timestamp);
```

Comment:
Any assert for the new field?

Reply:
Added test for the new field in CloudwatchRecordWorkerTest